clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -1281,17 +1259,11 @@ private void recordTopicFetchMetrics(String topic, int bytes, int records) {
                 Map<String, String> metricTags = Collections.singletonMap("topic", topic.replace('.', '_'));
 
                 bytesFetched = this.metrics.sensor(name);
-                bytesFetched.add(this.metrics.metricName("fetch-size-avg",
-                        this.metricGrpName,
-                        "The average number of bytes fetched per request for topic " + topic,
+                bytesFetched.add(this.metrics.metricInstance(metricsRegistry.topicFetchSizeAvg,
```

Comment:
Hmm.. I'm not sure what's the purpose of this refactoring: it seems we are just extracting the creating of the metricNames into a centralized class. Any benefits of doing this?